### ISSUE TITLE
Implement Sequelize with MySQL

### ISSUE THREAD

**Issue by Mayank0255** [Mayank0255/Stack-Overflow-Clone#17]
Convert the backend API models by using Sequelize instead of raw MySQL queries

**Comment by fabioalmeida100** [comment]
I can help with this issue @Mayank0255 Assigned to me :-)

**Comment by Mayank0255** [comment]
@fabioalmeida100 Done!!

All the best buddy!!

Contact me here or discord incase you have queries related to it

**Comment by Mayank0255** [comment]
@fabioalmeida100 

How's the work going on?

**Comment by fabioalmeida100** [comment]
Hello @Mayank0255 ! For now, only fork the project... I didn't can a time for begin convert the models and implement Sequelize ORM. But, I believe return to the code in the next week.

Do you need fast response for this issue?

**Comment by Mayank0255** [comment]
@fabioalmeida100 

Not really fast, was just hoping for a steady progress on it

**Comment by fabioalmeida100** [comment]
> @fabioalmeida100
> 
> Not really fast, was just hoping for a steady progress on it

I can't do steady progress on it... I thought that the support here was a part-time, because I work full time in my country. I can help with this project only in weekends.

Anyway, I'will leave you informed about my progress here with status report. If you want a person to work in steady way in business days, I can't at this moment...

**Status report:**

- In the next week, I want to work in the models for support the ORM and understand the complexity of the queries for plan the implemetation.

**Comment by Mayank0255** [comment]
@fabioalmeida100 ya no problem about that. I don't want daily progress. A weekly progress like you said is also fine with me.

Great, go ahead with the models

**Comment by fabioalmeida100** [comment]
@Mayank0255 maybe I will separate the queries from model with a implementation of the repository. So, I think this first step for convert models and queries for use Sequelize ORM.

For an easy test I think do a first PR with separation of queries of model and a the second PR with implementation of Sequelize is good... It will be more easy, I think in my opinion...

What do you think about this approach?

**Comment by Mayank0255** [comment]
@fabioalmeida100 ya this approach looks good to me too.

What I will do is create a new branch `dev` to which you can make the PRs, and then after the whole code is integrated and ready to be shown, then we will merge it with the master.

To go ahead with this process I can make two issues with the tasks you just mentioned and reference them here and after that you can make the PR linking to those two PRs separately.

What do you think about this?

**Comment by fabioalmeida100** [comment]
> @fabioalmeida100 ya this approach looks good to me too.
> 
> What I will do is create a new branch `dev` to which you can make the PRs, and then after the whole code is integrated and ready to be shown, then we will merge it with the master.
> 
> To go ahead with this process I can make two issues with the tasks you just mentioned and reference them here and after that you can make the PR linking to those two PRs separately.
> 
> What do you think about this?

Excellent @Mayank0255 ! I'm converting the models, here! In soon, my first PR in your project \o/

**Comment by Mayank0255** [comment]
@fabioalmeida100 Great, looking forward to it

**Comment by Amoghtech** [comment]
Is this issue still active?

**Comment by Mayank0255** [comment]
@Amoghtech yes it is active, is currently assigned to @fabioalmeida100 do you guys want to work together on this?

**Comment by Amoghtech** [comment]
Yes I'll be grateful

**Comment by Mayank0255** [comment]
@Amoghtech cool, communicate with fabio on where he is currently and till then you can start you work with the other issue I've assigned to you just now.

**Comment by fabioalmeida100** [comment]
> Is this issue still active?

Yes! I'm working on this @Amoghtech . So for now I don't need help in this issue :-)

**Comment by Mayank0255** [comment]
@Amoghtech okay so you can work on the othe issue assigned to you and after that you can work on a new issue which is available

**Comment by Mayank0255** [comment]
@fabioalmeida100 yesterday I have split up the front-end and back-end code in two repositories, the front-end code remains here, and the the backend code is being shifted to   [Stackoverflow-Clone-Backend](https://github.com/Mayank0255/Stackoverflow-Clone-Backend)
As the project is growing and more developers are joining in, the management is getting tougher and the number of issues they are facing are also increasing, this is the reason this has been done. It will also us to make the code quality even better and allow adding features more streamlined.

So, this issue and the 2 other sub-issues are more suitable to the backend repo, that's why I will be creating these same issues over there and closing them here. I request you to comment on the issues over there and kindly shift your code accordingly.

**Comment by Mayank0255** [comment]
Didn't know you can transfer issues, so will just do that
